packages/api/test/e2e/queries.e2e.ts

```diff
@@ -116,23 +116,23 @@ describe('e2e queries', () => {
   // });
 
   describe('Staking account derived query', () => {
-    // it('Gets staking account details', async done => {
-    //   const stashId = aliceStash.address;
-    //   const stakingAccount = await api.derive.staking.accountInfo(stashId);
-    //   expect(stakingAccount.accountId.toString()).toBe(stashId);
-    //   expect(stakingAccount.controllerId.toString()).toBe(alice.address);
-    //   expect(stakingAccount.nominators).toHaveLength(0); // Initially no nominators
-    //   expect(stakingAccount.rewardDestination.isStash).toBeTruthy();
-    //   expect(stakingAccount.stakers).toBeDefined();
-    //   expect(stakingAccount.stakingLedger.stash.toString()).toBe(stashId);
-    //   expect(stakingAccount.validatorPrefs[0]).toBe('commission');
-    //   expect(stakingAccount.validatorPrefs[1].toNumber()).toBe(0);
-    //   const stakingSessionDetails = await api.derive.session.keyInfo(stashId);
-    //   const session = '5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu';
-    //   expect(stakingSessionDetails.nextSessionKeys[0].toString()).toBe(session);
-    //   expect(stakingSessionDetails.sessionKeys[0].toString()).toBe(session);
-    //   done();
-    // });
+    it('Gets staking account details', async done => {
+      const stashId = aliceStash.address;
+      const stakingAccount = await api.derive.staking.accountInfo(stashId);
+      expect(stakingAccount.accountId.toString()).toBe(stashId);
+      expect(stakingAccount.controllerId.toString()).toBe(alice.address);
+      expect(stakingAccount.nominators).toHaveLength(0); // Initially no nominators
+      expect(stakingAccount.rewardDestination.isStaked).toBeTruthy();
```

Comment:
This should still be `isStash`, maybe type needs change?

Reply:
for some reason I guess it uses polkadot's type for staking module.. This is the only module with same name but different internals from polkadot... Need to find a way to override polkadot's staking types.. and may that's the reason the staking call ends up with **bond call gave - tx` between node and runtime: Error decoding field Call :: Staking.0)**
https://centralitydev.atlassian.net/browse/IND-351
will investigate in this..

Reply:
Ah yes, I'd say that's the reason for the decoding failure also

Reply:
added the missing type for staking.. However staking bond tx still gives a error... a better error now
2020-10-19 17:06:25        RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1010: Invalid Transaction: Inability to pay some fees , e.g. account balance too low
which means at least the decoding encoding seems fine..

Reply:
would look more on Staking related extrinsic errors, while working on staking ticket..